@here/harp-mapview/lib/ElevationWrapperThingy.ts

```diff
@@ -0,0 +1,11 @@
+import { HeightProvider } from "@here/harp-mapview/lib/HeightProvider";
+import { DataSource } from "./DataSource";
+import { ElevationRangeSource } from "./ElevationRangeSource";
+
+export class ElevationWrapperThingy {
+    constructor(
+        readonly terrainDataSource: DataSource,
+        readonly heightProvider: HeightProvider,
```

Comment:
Maybe we can combine the height provider and the elevation range source. At least we should be consistent and name everything either Elevation... or Height...

Reply:
Combining them simplifies the API (one thing to construct as opposed to two), however it would require that some methods use the visible tiles (getHeight / raycast) and some make a separate request (ElevationRangeSource), but it should be possible to make this clear in the documentation / function names.

Reply:
Okay, we can merge it later if we still want